openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -58,11 +61,10 @@ OpenAssessment.ResponseView.prototype = {
                 view.server.renderLatex($(stepID, view.element));
                 view.installHandlers();
                 view.setAutoSaveEnabled(true);
-
                 view.isRendering = false;
                 view.baseView.announceStatusChangeToSRandFocus(stepID, usageID, false, view, focusID);
                 view.announceStatus = false;
-
+                view.dateFactory.report();
             }
         ).fail(function() {
```

Comment:
I agree it's a little bit redundant. I attempted to get it to fire using oa_base.js (the 'base view' renderer) and couldn't get it to line up. The utils need to fire after rendering, and I was only able to get it to work in the individual views. The redundancy is to insure against instances where course staff has only chosen to render one step of the possible assessment steps (e.g. staff assessment only). This assures that the datefactory fires with the correctly associated element at all times.